soh/src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -824,8 +839,15 @@ void EnMb_ClubAttack(EnMb* this, PlayState* play) {
     s16 flamesUnused[] = { 20, 40, 0 };
     s16 relYawTarget[] = { -0x9C4, 0, 0xDAC };
 
-    Math_SmoothStepToS(&this->actor.shape.rot.y, relYawTarget[this->attack - 1] + this->actor.world.rot.y, 1, 0x2EE, 0);
-
+    // Rotate Club Moblin towards player in Enemy Randomizer because they're borderline
+    // useless otherwise in most scenarios.
```

Comment:
```suggestion
    // Rotate Club Moblin towards player in Enemy Randomizer because they're
    // borderline useless otherwise in most scenarios.
```